### ISSUE TITLE
File in use exception after filling properties

### ISSUE THREAD

**Issue by pun-ky** [neva-dev/gradle-fork-plugin#38]
```
> Task :props FAILED
Build failure
Execution failed for task ':props'.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':props'.
> java.nio.file.FileSystemException: C:\Users\krystian.panek\Projects\gradle-aem-multi\gradle.user.properties: Proces nie mo▒e uzyska▒ dost▒pu do pliku, poniewa▒ jest on u▒ywany przez inny proces.

```

why? when project is open in intellij, it could prevent file saving of 'gradle.user.properties' occassionally

to fix it:

a) close IntelliJ, rerun `gradlew props`
b) run `gradlew --stop` then rerun `gradlew props`

this affects only Windows machines

any ideas how to fix / avoid it?

**Comment by pun-ky** [comment]
more stack trace:

```
Caused by: java.nio.file.FileSystemException: C:\Users\ciapu\Projekty\gradle-aem-multi\gradle.user.properties: Proces nie mo┐e uzyskaŠ dostŕpu do pliku, poniewa┐ jest on u┐ywany przez inny proces.

        at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:92)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
        at java.base/sun.nio.fs.WindowsFileCopy.copy(WindowsFileCopy.java:164)
        at java.base/sun.nio.fs.WindowsFileSystemProvider.copy(WindowsFileSystemProvider.java:283)
        at java.base/java.nio.file.Files.copy(Files.java:1294)
        at com.neva.gradle.fork.config.FileHandler$copy$1.invoke(FileHandler.kt:21)
        at com.neva.gradle.fork.config.FileHandler$copy$1.invoke(FileHandler.kt:10)
        at com.neva.gradle.fork.config.FileHandler.perform(FileHandler.kt:120)
        at com.neva.gradle.fork.config.rule.CopyTemplateFilesRule.copyAndExpandTemplateFiles(CopyTemplateFilesRule.kt:41)
        at com.neva.gradle.fork.config.rule.CopyTemplateFilesRule.execute(CopyTemplateFilesRule.kt:32)
        at com.neva.gradle.fork.config.Config.execute(Config.kt:326)
        at com.neva.gradle.fork.config.Config.evaluate(Config.kt:315)
        at com.neva.gradle.fork.tasks.ConfigTask.evaluate(ConfigTask.kt:18)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:104)
        ... 97 more

```

**Comment by pun-ky** [comment]
to be released in 5.0.0
